services/device-registry-mongodb/src/test/java/org/eclipse/hono/deviceregistry/mongodb/service/MongoDBBasedDeviceManagementSearchDevicesTest.java

```diff
@@ -66,6 +71,8 @@ public void setup(final VertxTestContext testContext) {
                 mongoClient,
                 config,
                 new NoopTenantInformationService());
+        final DownstreamSenderFactory downstreamSenderFactoryMock = mock(DownstreamSenderFactory.class);
```

Comment:
is this used anywhere?

Reply:
See https://github.com/eclipse/hono/pull/2360#discussion_r554910673